src/amo/components/RatingManager/index.js

```diff
@@ -251,6 +288,25 @@ export class RatingManagerBase extends React.Component<InternalProps, State> {
               review={!this.isSignedIn() ? null : userReview}
             />
           </div>
+          <div
+            className={makeClassName(
+              'RatingManager-savedRating',
+              {
+                'RatingManager-savedRating-hidden':
+                  flashMessage !== STARTED_SAVE_RATING &&
+                  flashMessage !== SAVED_RATING,
+              },
+              {
+                'RatingManager-savedRating-withReview': Boolean(
+                  userReview.body,
+                ),
+              },
+            )}
+          >
+            {flashMessage === STARTED_SAVE_RATING
+              ? i18n.gettext('Saving')
+              : i18n.gettext('Saved')}
```

Comment:
I like how the mock spells it out exactly as *Star rating saved* so I think we should do that.

<img width="386" alt="screenshot 2018-09-11 12 07 59" src="https://user-images.githubusercontent.com/55398/45375697-64d51c80-b5bb-11e8-9582-3d121d8bc920.png">

The mock also calls for a checkbox icon but, you know, I think it would be nicer to just render a proper notice for this (which gets you a checkbox icon):

```js
<Notice type="success">{i18n.gettext('Saved...')}</Notice>
```

What do you think? The other UX mock already has this notice type for saving a review. I don't see why we can't use it in both places:

<img width="385" alt="screenshot 2018-09-11 12 10 11" src="https://user-images.githubusercontent.com/55398/45375810-b1b8f300-b5bb-11e8-91ee-6d51958c8d2f.png">